phys/module_physics_init.F

```diff
@@ -3418,8 +3416,9 @@ SUBROUTINE bl_init(STEPBL,BLDT,DT,RUBLTEN,RVBLTEN,RTHBLTEN,  &
            if(isfc .ne. 1)CALL wrf_error_fatal &
             ( 'module_physics_init: Use sf_sfclay_physics= 1 or 91 for this pbl option' )
            IF ((SF_URBAN_PHYSICS.eq.2).OR.(SF_URBAN_PHYSICS.EQ.3)) CALL wrf_error_fatal &
-            ( 'module_physics_init: use myj (option 2) or boulac (option 8) with BEP/BEM urban scheme' )
+            ( 'module_physics_init: use ysu (option 1), myj (option 2), or boulac (option 8) with BEP/BEM urban scheme' )
            CALL shinhonginit(RUBLTEN,RVBLTEN,RTHBLTEN,RQVBLTEN,&
+
```

Comment:
This is an unnecessary added blank line. I'd remove it, but I don't want to trigger a full regression test just for this.

Reply:
Sorry about that, Dave. Must have accidentally added that space when modifying the wrf_error_fatal print.

Reply:
@eahendricks
Eric,
This change could be handled entirely on github. However, go ahead and clone this repo, make the change, do the git add  - commit - push.